light-clients/ics07-tendermint/src/lib.rs

```diff
@@ -33,7 +33,7 @@ pub mod client_state;
 pub mod consensus_state;
 pub mod error;
 pub mod events;
-mod merkle;
+pub mod merkle;
```

Comment:
why this change?

Reply:
we need a method from the library to convert tendermint proof to merkle proof